request/request.go

```diff
@@ -87,12 +89,18 @@ func NewRequestClient() *Client {
 
 	headers := []Header{
 		{Key: "X-Amp-Client", Value: clientName},
-		{Key: "X-Amp-Client-Version", Value: versionInfo.Version},
 		{Key: "X-Amp-Client-Commit", Value: versionInfo.CommitID},
 		{Key: "X-Amp-Client-Branch", Value: versionInfo.Branch},
 		{Key: "X-Amp-Client-Build-Date", Value: versionInfo.BuildDate},
 	}
 
+	if versionInfo.Version != "" {
+		headers = append(headers, Header{
+			Key:   "X-Amp-Client-Version",
+			Value: versionInfo.Version,
+		})
+	}
```

Comment:
Local versions of amp compiled without goreleaser won't have this set, and there's really no point in sending a blank header.